tests/AblyFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Ably\LaravelBroadcaster\Tests;
+
+use Ably\AblyRest;
+use Ably\LaravelBroadcaster\AblyBroadcaster;
+
+/**
+ * Instantiates AblyRest objects
+ */
+class AblyFactory
```

Comment:
We should be using this class as a part of `ServiceProvider` somewhere right?

Reply:
You can check if it's being used somewhere in https://github.com/ably/ably-php-laravel or can it be reused somewhere?  If not it's fine ..

Reply:
Looks good 👍